hq/app/logging/Cloudwatch.scala

```diff
@@ -0,0 +1,48 @@
+package logging
+
+import com.amazonaws.services.cloudwatch.AmazonCloudWatchClientBuilder
+import com.amazonaws.services.cloudwatch.model.{Dimension, MetricDatum, PutMetricDataRequest, PutMetricDataResult, StandardUnit}
+import model.{AwsAccount, CredentialReportDisplay}
+
+import collection.JavaConverters._
+
+
+object Cloudwatch {
+
+  object DataType extends Enumeration {
+    val s3Total = Value("s3/total")
+    val iamCredentialsTotal = Value("iam/credentials/total")
+    val iamKeysTotal = Value("iam/keys/total")
+    val sgTotal = Value("securitygroup/total")
+  }
+
+  def logAsMetric[T](data: Seq[T], dataType: DataType.Value ) : Unit = {
+    for ((account: AwsAccount, result) <- data) {
```

Comment:
I think you could also write this as:

```
data.foreach{
 case(account:AWSAccount, Right(details: CredentialReportDisplay)) => 
             putMetric(account, dataType, details.humanUsers.length + details.machineUsers.length)
```

etc. though I'm not certain it will come out looking any cleaner so ignore me if it becomes a mess!

Reply:
I've tried it and it does look cleaner to me! Thanks